app/models/appointment.rb

```diff
@@ -19,7 +19,17 @@ class Appointment < ApplicationRecord
                            message: I18n.t("must_be_a_valid_file_format")},
             size: {less_than: Settings.size_file.megabytes,
                    message: I18n.t("should_be_less_than_5MB")}
+
   delegate :full_name, :room, to: :doctor, prefix: true
   delegate :full_name, to: :patient, prefix: true
-  scope :appointment_by_created_at, ->{order created_at: :desc}
+
+  scope :appointment_by_created_at, ->{order created_at: :asc}
```

Comment:
```suggestion
  scope :by_created_at, ->{order created_at: :asc}
```
k cần thêm appointment vào tên scope vì đang ở trong model appoiment rồi